Source/APIClient/Request.swift

```diff
@@ -18,32 +18,18 @@ public enum HTTPMethod: String {
 public protocol Request {
     static var baseURL: String { get }
     static var acceptHeader: String? { get }
-    static var authToken: String? { get set }
+    static var authToken: String? { get }
```

Comment:
Exactly, the conforming type can do whatever it wants on top of the requirements of the protocol, including providing more access to properties.